packages/backend/prisma/schema.prisma

```diff
@@ -96,12 +97,15 @@ model connections {
   appId                      String?
   schema_mapping_id          String?
   schema_mapping             schema_mapping? @relation(fields: [schema_mapping_id], references: [id])
+  accountId                  String?
+  accounts                   accounts?       @relation(fields: [accountId], references: [id])
   createdAt                  DateTime        @default(now())
   updatedAt                  DateTime        @default(now()) @updatedAt
 
   @@unique([tp_customer_id, t_id], name: "uniqueCustomerPerTenant")
-  @@unique([tp_id, t_id], name: "uniqueThirdPartyPerTenant")
+  @@unique([tp_id, t_id, accountId], name: "uniqueThirdPartyPerTenantPerAccount")
```

Comment:
Can this be `perEnvironment` instead of `perAccount` because there's already a relationship established between them?